### PR TITLE
HW1 with calculator

### PR DESCRIPTION
**Pull request by lyalchenko1995**
Commit1 for HW1 with calculator

### CODE REVIEW

File: ArseniiLialchenko.iml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="CheckStyle-IDEA-Module" serialisationVersion="2">
+    <option name="activeLocationsIds" />
+  </component>
+</module>
```
Comment (komyagin) [inline ArseniiLialchenko.iml]: This type of file should not to be here

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+com_crashlytics_export_strings.xml
+crashlytics.properties
+crashlytics-build.properties
+fabric.properties
+
+# Editor-based Rest Client
+.idea/httpRequests
+
+# Android studio 3.1+ serialized cache file
+.idea/caches/build_file_checksums.ser
```
Comment (komyagin) [inline .gitignore]: Add new blank line at the end

File: pom.xml [hunk 3]
```diff
@@ -4,13 +4,82 @@
...
+        </profile>
+        <profile>
+            <id>multAndDiv</id>
+            <properties>
+                <suite.xml.file>multAndDivTests.xml</suite.xml.file>
+            </properties>
+        </profile>
+    </profiles>
+
 </project>
```
Comment (komyagin) [inline pom.xml]: Add new blank line at the end

File: src/test/resources/suiteAddSubsTests.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<suite name="suiteAddSubsTests" verbose="1" >
+    <test name="Test" >
+        <groups>
+            <run>
+                <include name="additionalSubstraction"></include>
+            </run>
+        </groups>
+    </test>
+</suite>
```
Comment (komyagin) [inline src/test/resources/suiteAddSubsTests.xml]: Add new blank line at the end

File: src/test/resources/suiteMiltDivTests.xml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+<suite name="suiteMiltDivTests" verbose="1" >
+    <test name="Test" >
+        <groups>
+            <run>
+                <include name="multiplyDivision"></include>
+            </run>
+        </groups>
+    </test>
+</suite>
```
Comment (komyagin) [inline src/test/resources/suiteMiltDivTests.xml]: Add new blank line at the end

File: pom.xml [hunk 6]
```diff
@@ -4,13 +4,82 @@
...
+            <version>7.6.1</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>calculator</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
+            <scope>system</scope>
+            <systemPath>${pom.basedir}/calculator/calculator-1.0.jar</systemPath>
```
Comment (komyagin) [inline pom.xml]: Where is a calculator in the repository? How should I run your project without this dependency?

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,77 @@
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
```
Comment (komyagin) [inline .gitignore]: Could be simplified

File: pom.xml [hunk 8]
```diff
@@ -4,13 +4,82 @@
...
-        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.testng</groupId>
+            <artifactId>testng</artifactId>
+            <version>7.6.1</version>
```
Comment (komyagin) [inline pom.xml]: The dependency version should move to the properties section and use a variable for it

File: pom.xml [hunk 9]
```diff
@@ -4,13 +4,82 @@
...
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>calculator</groupId>
+            <artifactId>calculator</artifactId>
+            <version>1.0</version>
```
Comment (komyagin) [inline pom.xml]: The dependency version should move to the properties section and use a variable for it

File: pom.xml [hunk 10]
```diff
@@ -4,13 +4,82 @@
...
+            <systemPath>${pom.basedir}/calculator/calculator-1.0.jar</systemPath>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>3.2.0</version>
```
Comment (komyagin) [inline pom.xml]: The dependency version should move to the properties section and use a variable for it

**Review by komyagin — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 8 above)
Comment (lyalchenko1995) [reply pom.xml]: done

File: .gitignore (hunk 2 above)
Comment (lyalchenko1995) [reply .gitignore]: done

File: pom.xml (hunk 3 above)
Comment (lyalchenko1995) [reply pom.xml]: done

File: src/test/resources/suiteAddSubsTests.xml (hunk 4 above)
Comment (lyalchenko1995) [reply src/test/resources/suiteAddSubsTests.xml]: done

File: src/test/resources/suiteMiltDivTests.xml (hunk 5 above)
Comment (lyalchenko1995) [reply src/test/resources/suiteMiltDivTests.xml]: done

File: pom.xml (hunk 9 above)
Comment (lyalchenko1995) [reply pom.xml]: done

File: pom.xml (hunk 10 above)
Comment (lyalchenko1995) [reply pom.xml]: done

File: ArseniiLialchenko.iml (hunk 1 above)
Comment (lyalchenko1995) [reply ArseniiLialchenko.iml]: done

File: .gitignore (hunk 7 above)
Comment (lyalchenko1995) [reply .gitignore]: done

File: pom.xml (hunk 6 above)
Comment (lyalchenko1995) [reply pom.xml]: done

**Review by komyagin — APPROVED** [review]

**Pull request merged**